btree/btree.h

```diff
@@ -158,13 +158,15 @@ enum m0_btree_opflag {
  * credit to @accum.
  */
 M0_INTERNAL void m0_btree_create_credit(const struct m0_btree_type *bt,
-					struct m0_be_tx_credit *accum);
+					struct m0_be_tx_credit *accum,
+					m0_bcount_t nr);
```

Comment:
Update the comment above to include description for nr.

Reply:
Done